bin/ember-template-lint.js

```diff
@@ -134,6 +134,15 @@ function parseArgv(_argv) {
         describe: 'Output errors with source description',
         boolean: true,
       },
+      'working-directory': {
```

Comment:
My preference would to just name this `cwd` and remove the alias, but I know you disagree, so 🤷‍♂️

Reply:
Hehe, yes indeed. I much prefer `--working-directory` over a cryptic shorthand. However, shorthands are **perfect** for aliases. 😃